crates/rustc_codegen_spirv/src/symbols.rs

```diff
@@ -129,21 +129,21 @@ const BUILTINS: &[(&str, BuiltIn)] = {
         ("bary_coord_no_persp_nv", BaryCoordNoPerspNV),
         ("frag_size_ext", FragSizeEXT),
         ("frag_invocation_count_ext", FragInvocationCountEXT),
-        ("launch_id_nv", LaunchIdNV),
-        ("launch_size_nv", LaunchSizeNV),
-        ("world_ray_origin_nv", WorldRayOriginNV),
-        ("world_ray_direction_nv", WorldRayDirectionNV),
-        ("object_ray_origin_nv", ObjectRayOriginNV),
-        ("object_ray_direction_nv", ObjectRayDirectionNV),
-        ("ray_tmin_nv", RayTminNV),
-        ("ray_tmax_nv", RayTmaxNV),
-        ("instance_custom_index_nv", InstanceCustomIndexNV),
-        ("object_to_world_nv", ObjectToWorldNV),
-        ("world_to_object_nv", WorldToObjectNV),
+        ("launch_id_khr", BuiltIn::LaunchIdKHR),
+        ("launch_size_khr", BuiltIn::LaunchSizeKHR),
+        ("instance_custom_index_khr", BuiltIn::InstanceCustomIndexKHR),
+        ("ray_geometry_index_khr", BuiltIn::RayGeometryIndexKHR),
+        ("world_ray_origin_khr", BuiltIn::WorldRayOriginKHR),
+        ("world_ray_direction_khr", BuiltIn::WorldRayDirectionKHR),
+        ("object_ray_origin_khr", BuiltIn::ObjectRayOriginKHR),
+        ("object_ray_direction_khr", BuiltIn::ObjectRayDirectionKHR),
+        ("ray_tmin_khr", BuiltIn::RayTminKHR),
+        ("ray_tmax_khr", BuiltIn::RayTmaxKHR),
+        ("object_to_world_khr", BuiltIn::ObjectToWorldKHR),
+        ("world_to_object_khr", BuiltIn::WorldToObjectKHR),
+        ("hit_kind_khr", BuiltIn::HitKindKHR),
+        ("incoming_ray_flags_khr", BuiltIn::IncomingRayFlagsKHR),
         ("hit_t_nv", HitTNV),
```

Comment:
Shouldn't we just get rid of the NV raytracing stuff entirely in favor of KHR? 

There's little reason for anyone to use it for new software, KHR is the only future.

Reply:
Maybe? `HitTNV` seems to be the only builtin that wasn't brought over to the KHR extension. So there might be valid uses for it, but I don't know enough about it to be able to say one way or another.

Reply:
Supposedly it's just an alias of a variable that is in KHR:
https://github.com/KhronosGroup/GLSL/blob/ab928c80851b96f3889f1287d4ce0003a9bb86e4/extensions/nv/GLSL_NV_ray_tracing.txt#L657-L658

Reply:
Yeah it's nothing special and is only part of the NV extension, which can now be considered deprecated.

Reply:
Please just delete it along with the other NV stuff.

Reply:
Well, I've removed `HitTNV`, I'm going to leave any other pre-existing NV builtins however, as the rest are unrelated to ray tracing, and should be migrated separately as their own PRs.